### PR TITLE
Fixed CVE issue

### PR DESCRIPTION
**Pull request by chrisho**
**Problem:**
PR introduces changes to address CVE's

**Solution:**

**Related Issue:**
https://github.com/harvester/security/issues/29

**Test plan:**

**More info:**
```
2023-04-20T02:35:45.264Z        INFO    Secret scanning is enabled
2023-04-20T02:35:45.264Z        INFO    If your scanning is slow, please try '--security-checks vuln' to disable secret scanning
2023-04-20T02:35:45.264Z        INFO    Please see also https://aquasecurity.github.io/trivy/v0.34/docs/secret/scanning/#recommendation for faster secret detection
2023-04-20T02:35:55.896Z        INFO    Detected OS: suse linux enterprise server
2023-04-20T02:35:55.896Z        INFO    Detecting SUSE vulnerabilities...
2023-04-20T02:35:55.899Z        INFO    Number of language-specific files: 1
2023-04-20T02:35:55.899Z        INFO    Detecting gobinary vulnerabilities...

harvester/harvester-node-manager:dev (suse linux enterprise server 15.4)
========================================================================
Total: 0 (UNKNOWN: 0, LOW: 0, MEDIUM: 0, HIGH: 0, CRITICAL: 0)
```

### CODE REVIEW

File: package/Dockerfile [hunk 1]
```diff
@@ -1,6 +1,13 @@
 FROM registry.suse.com/bci/bci-base:15.4
 
 RUN zypper -n rm container-suseconnect && \
+    zypper install -y curl=7.79.1-150400.5.18.1 \
+      glibc=2.31-150300.46.1 \
+      libcurl4=7.79.1-150400.5.18.1 \
+      libopenssl1_1=1.1.1l-150400.7.31.2 \
+      libopenssl1_1-hmac=1.1.1l-150400.7.31.2 \
+      libzstd1=1.5.0-150400.3.3.1 \
+      openssl-1_1=1.1.1l-150400.7.31.2 && \
```
Comment (guangbochen) [inline package/Dockerfile:10]: IIUC, this will not be needed as the OS will bump those packages automatically. cc @bk201

Comment (bk201) [reply package/Dockerfile:10]: Yes, no need to specify the version. A new build should upgrade those packages.

**Review by guangbochen — APPROVED** [review]
LGTM, thanks

**Review by bk201 — APPROVED** [review]
lgtm!

**Pull request merged**